var/spack/repos/builtin/packages/namd/package.py

```diff
@@ -19,9 +19,8 @@ class Namd(MakefilePackage):
     manual_download = True
 
     version("develop", branch="master")
```

Comment:
```suggestion
    version("master", branch="master")
```

Reply:
Agreed for consistency. I've reinstated the 2.14b(1/2) versions (not to break old installations).